sources/packages/backend/workflow/BPMN/camunda-8/parttime-assessment-2021-2022.bpmn

```diff
@@ -773,6 +734,51 @@
     </bpmn:intermediateThrowEvent>
     <bpmn:sequenceFlow id="Flow_0e86y64" sourceRef="Event_1sg75ph" targetRef="Event_0ckfmt3" />
     <bpmn:sequenceFlow id="Flow_0kk6ogk" sourceRef="Event_0ckfmt3" targetRef="Gateway_1o496q2" />
+    <bpmn:intermediateThrowEvent id="Event_1iix0z3" name="calculate CSGD total limit">
+      <bpmn:extensionElements>
+        <zeebe:ioMapping>
+          <zeebe:output source="=min(&#10;	max(&#10;	    if (calculatedDataTotalFamilyIncome &#60;= dmnPartTimeAwardFamilySizeVariables.limitAwardCSGDIncomeCap)&#10;		then &#10;			dmnPartTimeAwardAllowableLimits.limitAwardCSGD2OrLessChildAmount * offeringWeeks&#10;		else&#10;			max(dmnPartTimeAwardAllowableLimits.limitAwardCSGD2OrLessChildAmount - ((calculatedDataTotalFamilyIncome - dmnPartTimeAwardFamilySizeVariables.limitAwardCSGDIncomeCap) * dmnPartTimeAwardFamilySizeVariables.limitAwardCSGD2OrLessChildSlope), 0) * offeringWeeks&#10;        ,&#10;        dmnPartTimeAwardAllowableLimits.limitAwardCSGD2OrLessChildAmount&#10;    ),&#10;	dmnPartTimeAwardAllowableLimits.limitAwardCSGDAmount&#10;)" target="federalAwardCSGDAmount" />
```

Comment:
Minor step 3 could be moved to this node to be in sync with others. 

![image](https://github.com/bcgov/SIMS/assets/54600590/33a17c18-485e-4655-9063-95404ab90379)